src/settings/whitelabel/WhitelabelSettingsViewer.js

```diff
@@ -141,7 +141,9 @@ export class WhitelabelSettingsViewer implements UpdatableSettingsViewer {
 		const onThemeChanged = (theme) => {
 			neverNull(this._whitelabelConfig).jsonTheme = JSON.stringify(theme)
 			update(neverNull(this._whitelabelConfig))
-			themeManager.updateCustomTheme(theme, false)
+			theme.themeId = assertNotNull(this._whitelabelDomainInfo).domain
+			// Make sure to not apply it always with realtime color change later
```

Comment:
What does this mean? Why are we making it permanent now?

Reply:
:thinking: